### ISSUE TITLE
application entry js file

### ISSUE THREAD

**Issue by basaran** [sveltejs/kit#2057]
### Describe the problem

Hello, I'm sorry if this is the wrong place to ask but I couldn't find anything in the docs and the discord channel.

Is there a way to execute some javascript code when the app gets initialized by the browser for the first time? Something like an `app.js`, `main.js` or a `client.js`.

### Describe the proposed solution

I would like to set some localStorage items only once. 

### Alternatives considered

Closest I could think of was to add the `js` to the `__layout.svelte`.

### Importance

would make my life easier

**Comment by benmccann** [comment]
There's another ticket asking for this somewhere, so I'm going to close this. Yes, `_layout` is the closest at the moment
